modules/user_accounts/php/edit_user.class.inc

```diff
@@ -293,102 +290,118 @@ class Edit_User extends \NDB_Form
             // START EXAMINER UPDATE
             // If examiner not in table add him,
             // otherwise update the radiologist field and get the current sites
-            $values = \Utility::nullifyEmpty($values, 'examiner_radiologist');
-            if (empty($examinerID)) {
-
+            if (!empty($ex_radiologist)
+                && !empty($ex_pending)
+                && !empty($ex_curr_sites)
+                && empty($examinerID)
+            ) {
+                print_r("radio, pending, curr site not empty + examiner empty");
+                $ex_radiologist = $ex_radiologist === 'Y' ? 1 : 0;
+                $values = \Utility::nullifyEmpty($values, 'examiner_radiologist');
                 $DB->insert(
                     'examiners',
                     array(
-                     'full_name'   => $values['Real_name'],
-                     'radiologist' => $ex_radiologist,
-                     'userID'      => $uid,
+                        'full_name'   => $values['Real_name'],
+                        'radiologist' => $ex_radiologist,
+                        'userID'      => $uid,
                     )
                 );
                 $examinerID = $examinerID = $DB->pselectOne(
```

Comment:
Is this supposed to be just `$examinerID = $DB->pselectOne(...)`, instead of `$examinerID = $examinerID = $DB->pselectOne(...)`